rosdep/python.yaml

```
@@ -4922,6 +4922,13 @@ python3-argcomplete:
   openembedded: [python3-argcomplete@meta-python]
   rhel: ['python%{python3_pkgversion}-argcomplete']
   ubuntu: [python3-argcomplete]
+python3-asn1tools-pip:
+  debian:
+    pip:
+      packages: [asn1tools]
+  ubuntu:
+    pip:
+      packages: [asn1tools]
```

Since this rule applies to all (supported) platforms, please use the OS-level wildcard for pip.

```suggestion
  '*':
    pip:
      packages: [asn1tools]
```